ydb/core/sys_view/common/schema.cpp

```diff
@@ -286,6 +286,13 @@ class TSystemViewResolver : public ISystemViewResolver {
         RegisterSystemView<Schema::TopPartitions>(TopPartitions1HourName);
 
         RegisterPgTablesSystemViews();
+
+        {
+            using namespace NAuth;
+            RegisterSystemView<Schema::AuthUsers>(UsersName);
+            RegisterSystemView<Schema::AuthGroups>(NAuth::GroupsName);
```

Comment:
Вижу, что в 268 строке есть другой `GroupsName`. Неужели в 291 строке `using namespace NAuth;` не помогает?
```suggestion
            RegisterSystemView<Schema::AuthGroups>(GroupsName);
```

Reply:
Да, там будет 

```
home/kungasc/ydb/ydb/core/sys_view/common/schema.cpp:293:52: error: reference to 'GroupsName' is ambiguous
  293 |             RegisterSystemView<Schema::AuthGroups>(GroupsName);
      |                                                    ^
/home/kungasc/ydb/ydb/core/sys_view/common/schema.h:28:22: note: candidate found by name lookup is 'NKikimr::NSysView::GroupsName'
   28 | constexpr TStringBuf GroupsName = "ds_groups";
      |                      ^
/home/kungasc/ydb/ydb/core/sys_view/common/schema.h:54:26: note: candidate found by name lookup is 'NKikimr::NSysView::NAuth::GroupsName'
   54 |     constexpr TStringBuf GroupsName = "auth_groups";
      |                          ^
```